fedmsg_meta_fedora_infrastructure/tests/github.py

```diff
@@ -1834,6 +1834,112 @@ class TestGithubRelease(Base):
         }
     }
 
+class TestGithubPageBuild(Base):
+    """ There exists `a service
+    <https://apps.fedoraproject.org/github2fedmsg>`_ to link the select github
+    repos of fedora contributors with the fedmsg bus.
+
+    Messages of *this* type are published whenever someone **builds a Github Page**.
+    """
+    expected_title = "github.page_build"
+    expected_subti = None
+    expected_link = "https://github.com/fedora-infra/fas3-api-haskell"
```

Comment:
Ooo.. can you make the ``expected_link`` be ``"http://fedora-infra.github.io/fas3-api-haskell/"``?  That's the link to the page that actually got built.